lib/Conversion/TritonGPUToLLVM/MakeRangeOpToLLVM.cpp

```diff
@@ -25,7 +26,28 @@ struct MakeRangeOpConversion
     auto elemTy = ty.getElementType();
     assert(elemTy.isInteger(32));
     Value start = createIndexAttrConstant(rewriter, loc, elemTy, op.getStart());
-    auto idxs = emitIndices(loc, rewriter, targetInfo, layout, ty, true);
+    std::optional<int> warpGroupStart;
+    if (!getWarpGroupStart(rewriter.getInsertionBlock())) {
```

Comment:
This is quite hacky

Reply:
This is work-around for limited capability of our partitioner that doesn't put `make_range` op into a `warp_group` region. We actually like Meta's partitioner, and hopefully we can transition to it, and this issue get resolved, and this work-around won't be needed.